### ISSUE TITLE
Error when updating package with no updates

### ISSUE THREAD

**Issue by tylermorganwall** [tylermorganwall/adjustedcranlogs#1]
```r
> adjustedcranlogs::adj_cran_downloads("adjustedcranlogs",from = "2017-11-23", to = "2018-03-01")
Error in listupdates[[package]] : subscript out of bounds
```

**Comment by tylermorganwall** [comment]
Fixed in a755a8dc467018b8272e9f420b809b0270a8897b

**Comment by jean997** [comment]
I still get this error with the latest version
